src/main/java/org/klodnicki/exception/MaximumBookBorrowedLimitException.java

```diff
@@ -0,0 +1,10 @@
+package org.klodnicki.exception;
+
+public class MaximumBookBorrowedLimitException extends Exception{
+
+    private static final String MESSAGE = "Maximum limit is reached. The reader cannot borrow more than 10 books.";
```

Comment:
You're hardcoding the same value here as before. Find a way to fix it, like you did it before.